app/forks.go

```diff
@@ -33,9 +33,9 @@ func (app *Evmos) ScheduleForkUpgrade(ctx sdk.Context) {
 
 	// handle mainnet forks with their corresponding upgrade name and info
 	switch ctx.BlockHeight() {
-	case v82.MainnetUpgradeHeight:
-		upgradePlan.Name = v82.UpgradeName
-		upgradePlan.Info = v82.UpgradeInfo
+	case v16.TestnetUpgradeHeight:
```

Comment:
NIT - Why are we removing the old ones? - we can just add a different case.

Reply:
Re-adding back the old one for reference

Reply:
@facs95 @Vvaradinov  be careful here because one is for mainnet and the other one is for testnet

Reply:
And above we changed that if it is mainnet, the function will return

Reply:
With that check you added @GAtom22 for mainnet  it should work for testnet now right? We will remove it afterwards before mainnet or reverse the check ?

Reply:
I don't understand what's the point of keeping it. This is adding unnecessary overhead for each of the blocks on mainnet. That upgrade height already passed
Is there a good reason to keep it?

Reply:
Yeah I guess it make sense we don't need the code itself but just a reference somewhere to what happened at this height in documentation for future reference.

Reply:
you will still have this code in previous versions